--- AJAXChat.php	2014-01-10 21:00:34.000000000 +0100
+++ AJAXChat.php	2015-05-08 13:14:01.411495107 +0200
@@ -760,7 +760,12 @@
 				case '/nick':				
 					$this->insertParsedMessageNick($textParts);
 					break;
-			
+
+				// Youtube:
+				case '/yt':				
+					$this->insertParsedMessageYT($textParts);
+					break;
+		
 				// Custom or unknown command:
 				default:
 					if(!$this->parseCustomCommands($text, $textParts)) {				
@@ -3322,5 +3327,34 @@
 		return $this->_allChannels;
 	}
 
+	// Youtube
+	function insertParsedMessageYT($textParts) {
+		if(count($textParts) == 2){
+			//Grabs the youtubeID
+			parse_str( parse_url( $textParts[1], PHP_URL_QUERY ), $my_array_of_vars );
+			if ($my_array_of_vars['v'] != '' ||$my_array_of_vars['v'] != null){
+				//ChatbotDisplay Message
+				$YTembedLink = 'https://www.youtube.com/embed/'.$my_array_of_vars['v'].'?rel=0&border=0&autoplay=1';
+
+				$fetch_title = file_get_contents("http://youtube.com/get_video_info?video_id=".$my_array_of_vars['v']);
+				parse_str($fetch_title, $data);
+				$YouTubetitle = stripslashes($data['title']);
+
+				$this->insertCustomMessage(
+					$this->getUserID(),
+					$this->getUserName(),
+					$this->getUserRole(),
+					$this->getChannel(),
+				$YouTubetitle.' '.$YTembedLink
+				);
+			}
+			else{
+				//ERROR not valid Link
+			}
+		}
+		else{
+			//ERROR not valid Link
+		}
+	}
 }
-?>
\ No newline at end of file
+?>
